civihr_employee_portal/features/node_export_files/onboarding_form.export

```diff
@@ -4828,8 +4828,8 @@ array(
         99999 => 'Emergency_Contacts',
       ),
       'locationTypes' => array(
-        1 => 'Home',
         2 => 'Work',
+        6 => 'Personal',
```

Comment:
Is this some fixed ID in the database?

Reply:
It is, but my changes from PCHR-3004 introduced a [class](https://github.com/compucorp/civihr-employee-portal/commit/127546737e522c4033c9ecc0b5e4316e384c0c49) that handles differences in IDs across systems and instead uses the name. 

The change on this line is part of that, it's a mapping of location type IDs to names, and then that mapping is used to find the new location type ID on the system where the node will be imported.